int16 first_int
int16 second_int
